wheels/tests/populate.cfm

```diff
@@ -385,6 +385,8 @@ FROM users u INNER JOIN galleries g ON u.id = g.userid
 <cfset local.raul = model("author").create(firstName="Raul", lastName="Riera")>
 <cfset local.andy = model("author").create(firstName="Andy", lastName="Bellenie")>
 <cfset local.adam = model("author").create(firstName="Adam", lastName="Chapman, Duke of Surrey")>
+<cfset local.adam = model("author").create(firstName="Tom", lastName="King")>
```

Comment:
Why am I adam? :@

Reply:
I thought we'd sound pretty good as Adams :)  I've fixed it now

Reply:
I can only dream of being an adam. One day.

Reply:
Join me.. It's bliss!